server/src/test/java/io/crate/lucene/NumericEqQueryTest.java

```diff
@@ -34,7 +35,9 @@ protected String createStmt() {
         return """
             create table n (
                 x numeric(18, 2),
-                y numeric(38, 2)
+                y numeric(38, 2),
+                xarr numeric(18, 2)[],
+                yarr numeric(38, 2)[]
```

Comment:
Looks like there aren't any specific tests related to the precision of `38`? If so, there seems no need to add the `yarr` column.